mswindows/osgeo4w/package.sh

```diff
@@ -234,6 +241,19 @@ if [ -n "$PACKAGE_PATCH" ]; then
     unix2dos etc/preremove/grass${PACKAGE_POSTFIX}.bat
 
     # copy dependencies (TODO: to be reduced)
+    blas_dll=../msys64/mingw64/bin/libblas.dll
```

Comment:
Is it needed to treat new dependecies separately? Why not include them to `DLL` variable (https://github.com/OSGeo/grass/blob/main/mswindows/osgeo4w/package.sh#L109)?

Reply:
Included dependencies in DLL variable in c29f606